test/tests/parallel/set2/spam_bounced.bats

```diff
@@ -0,0 +1,31 @@
+TEST_NAME_PREFIX='spam (Amavis):'
+CONTAINER_NAME='dms-test-spam_bounced'
+
+load "${REPOSITORY_ROOT}/test/helper/setup"
+load "${REPOSITORY_ROOT}/test/helper/common"
+
```

Comment:
```bash
# Can run tests in parallel?: No
# Shared static container name: TEST_NAME

# Test case
# ---------
# When SPAMASSASSIN_SPAM_TO_INBOX=0, spam messages must be bounced (rejected).
# SPAMASSASSIN_SPAM_TO_INBOX=1 is covered in `mail_spam_junk_folder.bats`.
# Original test PR: https://github.com/docker-mailserver/docker-mailserver/pull/1485
```

Is the information here not useful anymore?

- Looks like this test file had more than one test at some point, so the first few lines aren't useful.
- It does mention the other variation of the ENV being covered in a different test file, and presumably tracking down the original test contribution warranted adding the PR link for reference :sweat_smile: 

I am fine with dropping it if no other maintainer sees value in it. I think it was intended to provide some summary of tests at the top and perhaps some additional context to ease maintenance / review, especially on larger or more complicated test files.

Reply:
I can re-add this comment :)